docs/Documentation/Systems/Vermilion/modules.md

```diff
@@ -10,7 +10,7 @@ grand_parent: Systems
 # Vermilion Modules and Applications
 *This page describes how to activate and use the modules available on Vermilion. Modules are not available by default on the machine.  This page discusses how to enable them.*
 
-The page [Running](./running.md) describes running on Vermilion in more detail including a description of the hardware, partitions, simple build and run scripts and launching Vasp.
+The page [Running on Vermilion](./running.md) describes running on Vermilion in more detail including a description of the hardware, partitions, and simple build and run scripts. For performance recommendations and sample scripts for running specific applications, please see the [Applications](../../Applications/ansys.md) section. 
```

Comment:
I don't think you actually mean to specifically link to ansys here do you?
```suggestion
The page [Running on Vermilion](./running.md) describes running on Vermilion in more detail including a description of the hardware, partitions, and simple build and run scripts. For performance recommendations and sample scripts for running specific applications, please see specific applications in the [Applications](../../Applications) section. 
```

Reply:
I think it's a little confusing. I would just say see specific application under applications section. I updated my suggestion to reflect this, but if you think something else is best then I'll trust your judgment!